src/components/header/_header.scss

```diff
@@ -74,13 +81,26 @@
       }
     }
 
-    @include mq(xs, m) {
+    @include mq(xs, 739px) {
       line-height: 1.75 !important;
     }
 
     @include mq(xxs, xs) {
       line-height: 1.7 !important;
     }
+
+    //to allow for differences in size between welsh and english logos
```

Comment:
It looks like the Welsh logo needs replacing, the graphical part isn't the same size as the english version